src/crawlee/browsers/_browser_pool.py

```diff
@@ -262,13 +262,15 @@ async def _get_new_page(
     ) -> CrawleePage:
         """Internal method to initialize a new page in a browser using the specified plugin."""
         timeout = self._operation_timeout.total_seconds()
-        browser = self._pick_browser_with_free_capacity(plugin)
+        browser_controller = self._pick_browser_with_free_capacity(plugin)
```

Comment:
Renamed to avoid confusion as the **browser_controller.new_page** is different to **browser_controller._browser.new_page**.